<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>666</width>
    <height>571</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>31</y>
      <width>561</width>
      <height>461</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="searchBoxLayout">
     <item>
      <widget class="QTableView" name="tableView"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="btnAdd">
         <property name="text">
          <string>Добавить</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnRemove">
         <property name="text">
          <string>Удалить</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnEdit">
         <property name="text">
          <string>Редактировать</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="searchLineEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="searchButton">
         <property name="text">
          <string>Поиск</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>666</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Действия</string>
    </property>
    <addaction name="actionImportCSV"/>
    <addaction name="actionExportCSV"/>
    <addaction name="actionExit"/>
    <addaction name="actionImportJSON"/>
    <addaction name="actionExportJSON"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionImportCSV">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/new/prefix1/datei-download.png</normaloff>:/new/prefix1/datei-download.png</iconset>
   </property>
   <property name="text">
    <string>Импорт</string>
   </property>
  </action>
  <action name="actionExportCSV">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/new/prefix1/free-icon-export-4911806.png</normaloff>:/new/prefix1/free-icon-export-4911806.png</iconset>
   </property>
   <property name="text">
    <string>Экспорт</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/new/prefix1/free-icon-exit-2760599.png</normaloff>:/new/prefix1/free-icon-exit-2760599.png</iconset>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="actionImportJSON">
   <property name="text">
    <string>Импорт JSON</string>
   </property>
  </action>
  <action name="actionExportJSON">
   <property name="text">
    <string>Экспорт JSON</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
